Specification Registry/broadcast.json

```diff
@@ -0,0 +1,27 @@
+{
+    "1" : {
```

Comment:
I imagined this to be a ledger containing the artefact links of the published specification (created whenever a change is merged)

Reply:
Got it. Pretty easy to change it do that format. Might have to tweak the verifier nodes a bit but otherwise its pretty much done.